lte/gateway/c/core/oai/lib/hashtable/obj_hashtable_uint64.c

```diff
@@ -48,15 +48,7 @@
 #include "lte/gateway/c/core/oai/common/dynamic_memory_check.h"
 #include "lte/gateway/c/core/oai/lib/hashtable/hashtable.h"
 
-#if TRACE_HASHTABLE
-#define PRINT_HASHTABLE(hTbLe, ...)                                \
-  do {                                                             \
-    if (hTbLe->log_enabled) OAILOG_TRACE(LOG_UTIL, ##__VA_ARGS__); \
-  } while (0)
-#else
 #define PRINT_HASHTABLE(...)
```

Comment:
This is a no-op, we should clean this up.

Reply:
@themarwhal , we can leave the `#define PRINT_HASHTABLE(...)` for now as it will go away when we get rid of the hashtable library altogether.